playdate/sound.lua

```diff
@@ -68,18 +132,27 @@ function sampleplayer.meta:getRate()
   self.data:getPitch()
 end
 
+function sampleplayer.meta:setRateMod(signal)
+  error("[ERR] playdate.sound.sampleplayer:setRateMod() is not yet implemented.")
+end
+
 local fileplayer = {}
 playdate.sound.fileplayer = fileplayer
 fileplayer.meta = {}
 fileplayer.meta.__index = fileplayer.meta
 
+-- TODO: handle overloaded function signature (bufferSize)
 function fileplayer.new(path, bufferSize)
   -- TODO: is there a way to use bufferSize to control Love2D chunks?
   local sample = setmetatable({}, fileplayer.meta)
```

Comment:
@fletchmakes Ohh let's add an `@@ASSERT` here too in case someone tries setting bufferSize.

```suggestion
-- TODO: handle overloaded function signature (bufferSize)
function fileplayer.new(path, bufferSize)
  -- TODO: is there a way to use bufferSize to control Love2D chunks?
  @@ASSERT(bufferSize == nil, "[ERR] Parameter bufferSize is not yet implemented.")
  local sample = setmetatable({}, fileplayer.meta)
```